crates/swc_ecma_lints/src/rules/eqeqeq.rs

```diff
@@ -0,0 +1,100 @@
+use serde::{Deserialize, Serialize};
+use swc_common::{errors::HANDLER, Span};
+use swc_ecma_ast::*;
+use swc_ecma_visit::{noop_visit_type, Visit, VisitWith};
+
+use crate::{
+    config::{LintRuleReaction, RuleConfig},
+    rule::{visitor_rule, Rule},
+};
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum EqeqeqMode {
+    Always,
+    Never,
+}
+
+impl Default for EqeqeqMode {
+    fn default() -> Self {
+        Self::Always
+    }
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+pub struct EqeqeqConfig {
+    #[serde(default)]
+    mode: EqeqeqMode,
+}
+
+pub fn eqeqeq(config: &RuleConfig<EqeqeqConfig>) -> Option<Box<dyn Rule>> {
+    match config.get_rule_reaction() {
+        LintRuleReaction::Off => None,
+        _ => Some(visitor_rule(Eqeqeq::new(config))),
+    }
+}
+
+#[derive(Debug, Default)]
+struct Eqeqeq {
+    expected_reaction: LintRuleReaction,
+    mode: EqeqeqMode,
+}
+
+impl Eqeqeq {
+    fn new(config: &RuleConfig<EqeqeqConfig>) -> Self {
+        Self {
+            expected_reaction: *config.get_rule_reaction(),
+            mode: config.get_rule_config().mode,
+        }
+    }
+
+    fn emit_report(&self, span: Span, actual: &str, expected: &str) {
+        let message = format!("Expected '{}' and instead saw '{}'", expected, actual);
```

Comment:
I think `Use '===' instead` would be better.
This is not a parsing error, so it's not expected. I forgot pointing them in previous PRs.

Reply:
Yes